docker/Makefile

```diff
@@ -38,7 +40,7 @@ build: prep
 	@docker build --pull --build-arg vcs_ref=$(vcs_ref) --build-arg build_date=$(build_date) --build-arg version=$(version) --file puppet-agent-ubuntu/$(dockerfile) --tag puppet/puppet-agent-ubuntu:$(version) puppet-agent-ubuntu
 	@docker tag puppet/puppet-agent-ubuntu:$(version) puppet/puppet-agent:$(version)
 ifneq ($(IS_NIGHTLY),true)
-	@docker build --pull --build-arg vcs_ref=$(vcs_ref) --build-arg build_date=$(build_date) --build-arg version=$(version) --file puppet-agent-alpine/$(dockerfile) --tag puppet/puppet-agent-alpine:$(version) puppet-agent-alpine
+	@docker build --pull --build-arg vcs_ref=$(vcs_ref) --build-arg build_date=$(build_date) --build-arg version=$(version) --file puppet-agent-alpine/$(dockerfile) --tag puppet/puppet-agent-alpine:$(version) $(makefile_dir)/..
```

Comment:
I don't understand why you are changing this, can you give me more context?

Reply:
The alpine Dockerfile tries to copy files from the puppet-agent repository. Docker only has access to files in its context, which is that directory passed, so it needs to be passed the whole repository.